pkg/slo-controller/noderesource/plugins/batchresource/plugin.go

```diff
@@ -133,6 +133,14 @@ func (p *Plugin) Calculate(strategy *extension.ColocationStrategy, node *corev1.
 	nodeUsage := getNodeMetricUsage(nodeMetric.Status.NodeMetric)
 	systemUsed := quotav1.Max(quotav1.Subtract(nodeUsage, podAllUsed), util.NewZeroResourceList())
 
+	// System.Used = max(System.Used, Node.Anno.Reserved)
+	nodeReserved := util.GetNodeReservationFromAnnotation(node.Annotations)
+	for name, sysUsedQ := range systemUsed {
```

Comment:
How about using `quotav1.Max()`?
There miss some checks, e.g. if the returned `nodeReserved == nil`, and `node.Annotations == nil` (mainly for testing).